source/macos/AdaptiveCards/AdaptiveCards/Renderers/FactSetRenderer.swift

```diff
@@ -66,7 +68,12 @@ class FactSetRenderer: NSObject, BaseCardElementRendererProtocol {
         // Spacing between title and value in the horizontal Stack
         titleStack.trailingAnchor.constraint(equalTo: valueStack.leadingAnchor, constant: -10).isActive = true
         // Getting Max width from Host config if it exists
-        titleStack.widthAnchor.constraint(lessThanOrEqualToConstant: CGFloat(truncating: factsetConfig?.title.maxWidth ?? 150)).isActive = true
+        let maxAllowedWidth = CGFloat(truncating: factsetConfig?.title.maxWidth ?? 150)
+        if requiredWidth > maxAllowedWidth {
+            titleStack.widthAnchor.constraint(equalToConstant: CGFloat(maxAllowedWidth)).isActive = true
+        } else {
+            titleStack.widthAnchor.constraint(equalToConstant: CGFloat(requiredWidth + 2)).isActive = true
+        }
```

Comment:
```suggestion
        if requiredWidth > maxAllowedWidth {
            titleStack.widthAnchor.constraint(equalToConstant: maxAllowedWidth).isActive = true
        } else {
            titleStack.widthAnchor.constraint(equalToConstant: requiredWidth + 2).isActive = true
        }
```
Explicit `CGFloat` conversion is not required.